src/components/skip/skip.js

```diff
@@ -33,12 +34,26 @@ const COMPONENT_NAME = 'Skip';
  */
 @connect(mapStateToProps, bindActions(actions))
 class Skip extends Component {
+  /**
+   * on key down handler
+   *
+   * @param {KeyboardEvent} e - keyboard event
+   * @returns {void}
+   * @memberof Skip
+   */
+  onKeyDown = (e: KeyboardEvent): void => {
```

Comment:
Space / enter also triggers pause / play on the player. Doesn't this logic conflict with the interaction with the player ?